.github/workflows/precommit.yml

```diff
@@ -1,19 +1,29 @@
-name: Gradle Precommit
+name: Gradle Precommit and Asssemble
 on: [pull_request]
 
 jobs:
   precommit:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [windows-latest, macos-latest]  # precommit on ubuntu-latest is run as a part of the gradle-check workflow
+        os: [ubuntu-latest, windows-latest, macos-latest]  # precommit on ubuntu-latest is run as a part of the gradle-check workflow
```

Comment:
Can probably remove the "# precommit on ubuntu-latest is run as a part of the gradle-check workflow" comment now. It was meant to explain why ubuntu wasn't included in this list.

Reply:
Certainly, thanks @andrross !